src/main/java/emissary/output/DropOffUtil.java

```diff
@@ -712,52 +712,52 @@ public String makeDotFile(final String fullName) {
     /**
      * Get the file type from the metadata or the form string passed in
      *
-     * @param metaData map of payload metadata
+     * @param bdo IBaseDataObject
      * @return the file type (also added to metaData if not present)
      */
-    public String getFileType(final Map<String, String> metaData) {
```

Comment:
We should retain this old method signature and annotate it as `@Deprecated` to prevent upstream build failures.